src/post/post.entity.ts

```diff
@@ -81,6 +90,7 @@ export class Post extends BaseEntity {
         public readonly inReplyTo: number | null = null,
         public readonly threadRoot: number | null = null,
         private readonly readingTimeMinutes: number | null = null,
+        attachments: PostAttachment[] | null = null,
```

Comment:
If instead we do `public readonly attachments: PostAttachment[]` 

Then I think we save a bunch of code, we don't need the public member on the class above on line 73, and we don't need the assignment on line 112. The repository is already doing the mapping anyway, so we don't need to do it here!

That also means that `getPostAttachments` can return `Promise<PostAttachment[]>` and simplifies the implementation there, too!